cinema/tests/test_movie_api.py

```diff
@@ -10,6 +10,7 @@
 from rest_framework import status
 
 from cinema.models import Movie, MovieSession, CinemaHall, Genre, Actor
```

Comment:
if you got more than 2 imports in one line - best practice is write every import from newline ->
from something import(
first,
second,
third
)